Source/JavaScriptCore/jit/JITThunks.cpp

```diff
@@ -207,12 +207,12 @@ void JITThunks::finalize(Handle<Unknown> handle, void*)
     }
 }
 
-NativeExecutable* JITThunks::hostFunctionStub(VM& vm, TaggedNativeFunction function, TaggedNativeFunction constructor, const String& name)
+NativeExecutable* JITThunks::hostFunctionStub(VM& vm, TaggedNativeFunction function, TaggedNativeFunction constructor, ImplementationVisibility implementationVisibility, const String& name)
```

Comment:
Since `ImplementationVisibility` is added to the argument, it needs to be included in JITThunks' hashtable equality. See `WeakNativeExecutableHash` etc.